lib/muffin_man/notifications/v1.rb

```diff
@@ -0,0 +1,63 @@
+module MuffinMan
+  module Notifications
+    require "json"
+    require "sp_api_helpers"
+    class V1 < SpApiClient
+
+      NOTIFICATION_PATH = "/notifications/v1"
+
+      def create_destinations(arn, params={})
+        params = params.transform_keys{|key| key.to_s}
+        @local_var_path = "#{NOTIFICATION_PATH}/destinations"
+        destination_params = {"resourceSpecification": {"sqs": {"arn": arn}}, "marketplaceIds": params["marketplace_ids"]}
```

Comment:
Is marketplaceIds a parameter for this endpoint? I'm not seeing it in the docs, but I could be missing something

Reply:
It is required parameter in old MWS apis, and in SP-API for destinations I did't find marketplaceIds filter so I've added optionally. Anyway removed it